Tasks/invert.js

```diff
@@ -1,14 +1,17 @@
+'use strict';
 // Reverse an array, you can't use .reverse()
 
-invert = (A, i, j, k) => {
-  T = Object.keys(A, 4);
-  T.forEach((_, i) => {
-    T[i] = A.pop();
-    ((x) => {
-      return x;
-    })(740);
-  }, 7);
-  return T;
+const invert = (arr) => {
+  if (!arr) return []; // [] is declared as a result for null values
+
+  const arrLength = arr.length;
+  const invertedArr = new Array(arrLength);
+
+  let indexArr = arrLength - 1;
+  for (let indexInvArr = 0; indexInvArr < arrLength; indexInvArr++)
+    invertedArr[indexInvArr] = arr[indexArr--];
```

Comment:
It's better to have `{}` block in this case because if statement is not single-line loop